### PR TITLE
Fix 146

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

File: code/init_mooc_nb.py [hunk 1]
```diff
@@ -118,7 +119,11 @@ def init_notebook():
     except NameError:
         pass
 
-    print('Populated the namespace with:\n' + ', '.join(__all__))
+    print('Populated the namespace with:\n' +
+        ', '.join(init_mooc_nb) +
+        ', edx_components.'.join(edx_components.__all__) +
+        ', functions.'.join(functions.__all__))
```
Comment (akhmerov) [inline code/init_mooc_nb.py]: But this isn't true: we aren't populating namespace with `functions.bla`, rather just `bla`.

**Comment by akhmerov** [comment]
Please make PR's topical: most of the commits here don't relate to fixing #146.

**Comment by basnijholt** [comment]
Now it's better.

I accidentally branched from week7 instead of master, fixed now.

File: code/init_mooc_nb.py [hunk 2]
```diff
@@ -165,4 +172,4 @@ def init_notebook():
 
     # Patch a bug in holoviews
     from patch_holoviews import patch_all
-    patch_all()
+    patch_all()
```
Comment (akhmerov) [inline code/init_mooc_nb.py]: Forgot a newline :D

**Comment by akhmerov** [comment]
After you've added the newline go ahead and merge.

**Pull request merged**